Docs/source/visualization/ascent.rst

```diff
@@ -176,3 +176,175 @@ When the trigger condition is meet, ``cycle() == 300``, the actions in :code:`tr
               db_name: "cinema_out"
 
 You can view the Cinema Database result by opening :code:`cinema_databases/cinema_out/index.html`.
+
+
+Replay
+-------------------------
```

Comment:
`.rst` files need those aligned, unfortunately:
```suggestion
Replay
------
```

Reply:
1. You want me move all PR#1332 commit files to ./Docs/src/visualization/, and combine PR#1332 to PR#1331?
2. What do you mean "save the input file alongside the regular examples"? where should I put the input file?
3. I'll remove the bsub file
4. please show me how to croo-link the png file.
Thanks!